demisto_sdk/__main__.py

```diff
@@ -1029,6 +1031,57 @@ def init(**kwargs):
     return 0
 
 
+# ====================== generate-context ====================== #
+@main.command()
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    "-i", "--input", help="Path of the yml file (will change it in-place).",
+    required=True)
+@click.option(
+    "-e", "--examples",
+    help="Integrations: path for file containing command examples."
+         " Each command should be in a separate line."
+         " Scripts: the script example surrounded by quotes."
+         " For example: -e '!ConvertFile entry_id=<entry_id>'")
```

Comment:
If scripts are not supported yet, adding this might confuse.
BTW. Any reason apart from time that scripts aren't supported?

Reply:
Time at the moment.